pages/admin/company.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+  <div class="content">
+    <h2>Editar informações</h2>
+    <form @submit.prevent="check" novalidate>
+      <!-- Company name -->
+      <label for="name">Nome da empresa</label>
+      <input v-model="name" name="name" type="text" />
+      <p class="error">{{ errors.nameError | first }}</p>
+      <!-- Company email -->
+      <label for="email">Email de cadastro</label>
+      <input v-model="email" name="email" type="email" />
+      <p class="error">{{ errors.emailError | first }}</p>
+      <!-- Company phone -->
+      <label for="phone">Telefone</label>
+      <input v-model="phone" name="phone" type="number" />
+      <!-- Address fieldset -->
+      <fieldset>
+        <label for="address">Endereço</label>
+        <!-- Company CEP -->
+        <label for="cep">CEP</label>
+        <input v-model="cep" name="cep" type="text" />
+        <p class="error">{{ errors.cepError | first }}</p>
+        <!-- Company state -->
+        <label for="state">Estado</label>
+        <select>
+          <option value=""></option>
```

Comment:
Please, use `v-for` importing the state data from other file to make this file cleaner.